beater/api/intake/handler.go

```diff
@@ -37,7 +37,7 @@ import (
 var (
 	// MonitoringMap holds a mapping for request.IDs to monitoring counters
 	MonitoringMap = request.DefaultMonitoringMapForRegistry(registry)
-	registry      = monitoring.Default.NewRegistry("apm-server.server", monitoring.PublishExpvar)
+	registry      = monitoring.Default.NewRegistry("apm-server.server")
```

Comment:
What's the purpose of removing this option?

Reply:
Because they're reported by the monitoring.Do call. If we leave the option in, they end up being reported twice: once by monitoring.Do and once by expvar.Do.